protocol/v2/ssv/runner/committee.go

```diff
@@ -418,44 +418,52 @@ func (cr *CommitteeRunner) ProcessPostConsensus(logger *zap.Logger, signedMsg *t
 			}
 		}
 	}
+
+	var submissionStart time.Time
```

Comment:
Can we do `submissionStart := time.Now()` in both `if`s and delete this line?

Reply:
good idea, done.